src/neo/SmartContract/InteropService.Contract.cs

```diff
@@ -46,7 +41,7 @@ private static bool Contract_Create(ApplicationEngine engine)
                 if (contract != null) return false;
                 contract = new ContractState
                 {
-                    Guid = GetDeterministicGuid((Transaction)engine.ScriptContainer, engine.SyscallCounter),
+                    Id = engine.Snapshot.CurrentContractId + 1,
```

Comment:
```suggestion
                    Id = engine.Snapshot.CurrentContractId,
```

Reply:
`CurrentContractId `represents the latest contract number that has been created. If you need to create a new contract, you should add 1

Reply:
I think it should represent the next available id for a new contract. Because the first id is `0`.

Reply:
Ok,I got it.

Reply:
https://github.com/doubiliu/neoUT/pull/1

Reply:
@shargon, in your review you still proposes the next ID, right?

Reply:
Yes but we can change it.